### DISCUSSION TITLE
Can i automatically create a new autoComplete when i press a button

### DISCUSSION THREAD

**Discussion by dsamhabe** [TarekRaafat/autoComplete.js#409] [Q&A]
Hey guys, i have a input table where i enter product names and i want to have an Auto complete for this product field in my input table. With a button i can add a new row that is exactly like the first row but the auto complete doesn't get added to the table as you can see in the image i added. The one input field with the different styling has auto Complete but the other ones don't have it when i click the add row button.
Can you help me how to add an auto complete for this field no matter how much rows i add?
Any help would be much appreciated
![image](https://github.com/TarekRaafat/autoComplete.js/assets/117641107/cd4ba0f4-68ff-4dc0-8618-2ed8865ccb9f)

**Comment by evansharp** [comment]
I have a similar need. I can't figure out how to reinitialize or destroy/recreate the AutocompelteJS object so that it attaches to new DOM elements.

Anyone?
